app/src/main/java/com/example/newsportal/app/newsdetail/NewsDetailPopup.kt

```diff
@@ -0,0 +1,70 @@
+package com.example.newsportal.app.newsdetail
+
+import android.content.Context
+import android.content.res.Resources
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import android.widget.PopupWindow
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.bumptech.glide.Glide
+import com.example.newsportal.R
+import com.example.newsportal.databinding.PopupNewsDetailBinding
+import com.example.newsportal.domain.model.Article
+import com.example.newsportal.utils.toFormat
+
+class NewsDetailPopup(
+    width: Int,
+    height: Int,
+    binding: PopupNewsDetailBinding,
+    listener: NewsDetailPopupListener,
+    article: Article
+) : PopupWindow(binding.root, width, height, true) {
+
+    init {
+        binding.apply {
+            topAppBar.title = ""
+            topAppBar.setNavigationOnClickListener { listener.onBackButtonClicked() }
+            topAppBar.setOnMenuItemClickListener { menuItem ->
+                if (menuItem.itemId == R.id.browser) {
+                    menuItem.isEnabled = false
```

Comment:
Perhaps this action is not suitable in this case, since we return from the browser to this popup. And because of this we will not be able to open the browser again, because the button will be blocked.